native/machine-coding/atm-simulator/styles.css

```diff
@@ -0,0 +1,27 @@
+body {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    height: 100vh;
+    margin: 0;
+    background-color: #f0f0f0;
+}
```

Comment:
Body styles can be removed. Applying flex on body will not render the navbar properly I guess. **Do test in your local**.